bubble_sort/Java/BubbleSort.java

```diff
@@ -0,0 +1,58 @@
+/*
+* I have converted the code of bubble sort written in C++ into Java with some little modifications,
+* C++ code: https://github.com/NITSkmOS/Algorithms/blob/master/bubble_sort/Cpp/bubble_sort.cpp
+* */
+
+import java.util.ArrayList;
+
+public class BubbleSort
+{
+    private ArrayList<Integer> arr;
+
+    public BubbleSort(ArrayList<Integer> arr)
+    {
+        this.arr = arr;
+        bubbleSort();       // Main bubble sort algorithm is called
+    }
+
+    // Wrapper class is used so that values are changed inside the function
+    private void swap(int j)
+    {
+        int temp = arr.get(j);
+	    arr.set(j, arr.get(j + 1));
+	    arr.set(j+1, temp);
```

Comment:
Line contains following spacing inconsistencies:
- Tabs used instead of spaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpqa3_fkic/bubble_sort/Java/BubbleSort.java
+++ b/tmp/tmpqa3_fkic/bubble_sort/Java/BubbleSort.java
@@ -20,7 +20,7 @@
     {
         int temp = arr.get(j);
 	    arr.set(j, arr.get(j + 1));
-	    arr.set(j+1, temp);
+        arr.set(j+1, temp);
     }
 
     private void bubbleSort()
```